pages/index.js

```diff
@@ -84,6 +86,7 @@ export default function Page() {
               </div>
             </a>
           </Link>
+					<Searchbar books = { books }/>
```

Comment:
don't space separate things in html tags.

Reply:
fixed